apps/condo/domains/common/components/tasks/TasksContextProvider.tsx

```diff
@@ -51,8 +52,9 @@ const TasksContextProvider: React.FC<ITasksContextProviderProps> = ({ preloadedT
     // NOTE: Initial state of `tasks` cannot be initialized with `preloadedTaskRecords` prop, because a hook in parent component that loads tasks
     // will rerender this component as data comes, so, the state will not be reinitialized.
     const [tasks, setTasks] = useState<ITaskTrackableItem[]>([])
-    // Because it is impossible to initial assign it to `tasks` state, handle it separately
-    const [latestUpdatedTask, setLatestUpdatedTask] = useState<ITaskTrackableItem>()
+    // Timestamp of latest update in tasks without changing length of trackable tasks array
+    // Used to trigger `useEffects` when tasks are updated or deleted. Relying on length of tasks is not enough in effects
+    const [latestUpdate, setLatestUpdate] = useState<number>()
```

Comment:
Maybe it would be better to use the force rerender pattern for functional components here?
It could be done like this
```js
const [, forceUpdate] = useReducer(x => x + 1, 0)
```